### PR TITLE
[MLMC] Power sums computation

### PR DESCRIPTION
**Pull request by riccardotosi**
This PR updates the compilation after #6002.
Additionally, it uploads a process updating power sums at each time step, for a given variable.

### CODE REVIEW

File: applications/MultilevelMonteCarloApplication/MultilevelMonteCarloApplication.py [hunk 1]
```diff
@@ -2,10 +2,10 @@
 from __future__ import print_function, absolute_import, division
 
 # Application dependent names and paths
-import KratosMultiphysics
-import KratosMultilevelMonteCarloApplication
-application = KratosMultilevelMonteCarloApplication.KratosMultilevelMonteCarloApplication()
+import KratosMultiphysics as KM
+from KratosMultilevelMonteCarloApplication import *
```
Comment (RiccardoRossi) [inline applications/MultilevelMonteCarloApplication/MultilevelMonteCarloApplication.py:6]: the import WITHOUT * would be better practice...

**Comment by roigcarlo** [comment]
As far as i am concern, the compilation part its ok :+1:. Can't tell about the rest

File: applications/MultilevelMonteCarloApplication/custom_statistics/power_sums_statistics.cpp [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        if (KratosComponents<Variable<double>>::Has(mReferenceVariable)) {
+            const auto& r_reference_var = KratosComponents<Variable<double>>::Get(mReferenceVariable);
+            // Iterate over the nodes
+            #pragma omp parallel for
+            for(int i_node = 0; i_node < static_cast<int>(number_nodes); ++i_node) {
+                auto it_node = mrModelPart.NodesBegin() + i_node;
+                // Retrieve current time step local variable
+                double variable_current = it_node->GetSolutionStepValue(r_reference_var);
+                // Retrieve power sums
+                const double S1_old = it_node->GetValue(POWER_SUM_1);
```
Comment (RiccardoRossi) [inline applications/MultilevelMonteCarloApplication/custom_statistics/power_sums_statistics.cpp]: you could shorten this a lot by doing

~~~cpp
it_node->GetValue(POWER_SUM_1) += std::pow(variable_current,1);
~~~

**Review by RiccardoRossi — COMMENTED** [review]
i am ok about the PR apart for a minor comment about how to make the code shorter

File: applications/MultilevelMonteCarloApplication/MultilevelMonteCarloApplication.py (hunk 1 above)
Comment (RiccardoRossi) [reply applications/MultilevelMonteCarloApplication/MultilevelMonteCarloApplication.py:6]: ok, wrong comment, this is the app importer

File: applications/MultilevelMonteCarloApplication/custom_statistics/power_sums_statistics.cpp (hunk 2 above)
Comment (riccardotosi) [reply applications/MultilevelMonteCarloApplication/custom_statistics/power_sums_statistics.cpp]: Done!

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**